_data/projects/leaflet.yml

```diff
@@ -0,0 +1,10 @@
+name: Leaflet
+desc: JavaScript library for mobile-friendly interactive maps
+site: http://leafletjs.com
+tags:
+- Javascript
+- Maps
+- GIS
+upforgrabs:
+  name: easy-fix help-wanted
```

Comment:
Well, there might be an `easy fix` bug which one of the hardcore developers wants to work with, and there might be a `help wanted` bug which just tries to find a contributor with the right browser configuration or hardware.

If I'd have to choose between the two of them, I'd probably say "keep easy fix".

Shall I amend this PR?